ui/src/app/pages/register/register.component.html

```diff
@@ -156,72 +156,74 @@ <h1 id="title" class="title">Create an account</h1>
 
         <div class="form-control-group">
           <label class="label" for="input-re-password">Repeat password:</label>
-          <input nbInput
+          <input #rePass="ngModel"
                  [(ngModel)]="user.confirmPassword"
-                 #rePass="ngModel"
+                 [attr.aria-invalid]="rePass.invalid && rePass.touched ? true : null"
+                 [required]="getConfigValue('forms.validation.password.required')"
+                 [status]="rePass.dirty ? (rePass.invalid || password.value != rePass.value  ? 'danger' : 'success') : 'basic'"
                  [type]="getInputType()"
+                 fieldSize="large"
+                 fullWidth
                  id="input-re-password"
                  name="rePass"
-                 placeholder="Confirm Password"
-                 fullWidth
-                 fieldSize="large"
-                 [status]="rePass.dirty ? (rePass.invalid || password.value != rePass.value  ? 'danger' : 'success') : 'basic'"
-                 [required]="getConfigValue('forms.validation.password.required')"
-                 [attr.aria-invalid]="rePass.invalid && rePass.touched ? true : null">
-          <span nbSuffix
+                 nbInput
+                 placeholder="Confirm Password">
+          <span (click)="toggleShowPassword()"
+                class="password-icon"
                 ghost
-                (click)="toggleShowPassword()"
-                class="password-icon">
-          <nb-icon [icon]="showPassword ? 'eye-outline' : 'eye-off-2-outline'"
-                   pack="eva"
-                   [attr.aria-label]="showPassword ? 'hide password' : 'show password'">
+                nbSuffix>
+          <nb-icon [attr.aria-label]="showPassword ? 'hide password' : 'show password'"
+                   [icon]="showPassword ? 'eye-outline' : 'eye-off-2-outline'"
+                   pack="eva">
           </nb-icon>
         </span>
           <ng-container *ngIf="rePass.invalid && rePass.touched">
-            <p class="caption status-danger mb-1" *ngIf="rePass.errors?.required">
+            <p *ngIf="rePass.errors?.required" class="caption status-danger mb-1">
               Password confirmation is required!
             </p>
-            <p class="caption status-danger mb-1" *ngIf="password.value != rePass.value && !rePass.errors?.required">
+            <p *ngIf="password.value != rePass.value && !rePass.errors?.required" class="caption status-danger mb-1">
               Password does not match the confirm password.
             </p>
           </ng-container>
         </div>
 
         <ps-click-wrap
-          *ngIf="_isProduction && _psEnabled"
-          accessId="{{_sid}}"
-          groupKey="{{_groupKey}}"
-          signerIdSelector="input-name"
-          confirmationEmail="1">
+          *ngIf="_psEnabled"
```

Comment:
@iatdaitan I needed to change the signer ID to email, in order to be able to send the email information, since `custom_data` param. only holds first_name, last_name and company_name; see [this page for ref](https://clickwrap-developer.ironcladapp.com/reference/url-parameters-for-send)

Reply:
Got it, thank you.